laceworksdk/http_session.py

```diff
@@ -296,7 +297,8 @@ def get_pages(self, uri, params=None, **kwargs):
             try:
                 response_json = response.json()
                 next_page = response_json.get("paging", {}).get("urls", {}).get("nextPage")
-            except Exception:
+            except json.JSONDecodeError as e:
+                logger.error(f"Failed to decode response from Lacework as JSON: {e}\nResponse text: {response.text}")
```

Comment:
line too long, can you split it

Reply:
also you can include `exc_info=True` and then you don't need to include the `as e` and `{e}` inside the logger, it will include the trace route and debug information.

Can just do the response text and include that

Reply:
done